src/yamlFileCommands.ts

```diff
@@ -19,7 +19,8 @@ export class YamlFileCommands {
         let message: string;
 
         if (!document
-            || !(document.fileName.endsWith('.yaml') || document.fileName.endsWith('.json'))) {
+            || !(document.fileName.endsWith('.yaml') || document.fileName.endsWith('.yml')
+            || document.fileName.endsWith('.json'))) {
             message =
                 '\'OpenShift: Create\' command requires a .yaml or a .json file opened in editor.';
```

Comment:
The message text probably worth to be changed too in order to reflect the change. WDYT?

Reply:
I agree. I've updated the message text.